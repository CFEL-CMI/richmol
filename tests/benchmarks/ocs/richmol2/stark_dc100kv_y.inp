
tdse_init(t_start=0.0, t_end=0.0, dt=0.01, units="ps", solver="split_zhexpv", accur=1.0e-12, no_krylov=10)

fortran_tdse(flog="fortran.log")

path = "../cmi-richmol_watie/"
h0 = States(path + "ocs_energies_j0_j30.rchm",{'emax':60000,'fmax':30,'fmin':0,'mmin':-30,'mmax':30,'dm':1,'df':1})
mu = Tensor(path + "ocs_matelem_mu_j<j1>_j<j2>.rchm", rank=1, ind_sym="none", units="au", me_thresh=1.0e-12)

edc = Field(form=["0","edc0","0"], params={"edc0":100000}, units="V/cm")

ham = -1.0*mu*edc

eigen_run(h0, ham, {"print_rate":1,"file":"stark_energies_100000.0","print_coef":"wavefunc"}, state_ind=range(0,1))

