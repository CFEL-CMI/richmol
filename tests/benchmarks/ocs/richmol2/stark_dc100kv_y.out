
All output from the Fortran TDSE module will be printed into file: fortran.log


Initialize tdse module

   initial time: 0.0 ps
   terminal time: 0.0 ps
   time step: 0.01 ps

   Use Planck constant for time units ps and energy units cm^-1, hbar = 5.30883730500664

   Initialize grid of time-points
   Number of points: 1

   TDSE solver: split_zhexpv
   Requested accuracy of solution: 1e-12
   Number of Krylov vectors: 10


Initialize field-free basis

   States file: ../cmi-richmol_watie/ocs_energies_j0_j30.rchm
   State filters: {'emax': 60000, 'fmax': 30, 'fmin': 0, 'mmin': -30, 'mmax': 30, 'dm': 1, 'df': 1}

   selected F quanta: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30]

         F   no.levels
       0.0        1
       1.0        1
       2.0        1
       3.0        1
       4.0        1
       5.0        1
       6.0        1
       7.0        1
       8.0        1
       9.0        1
      10.0        1
      11.0        1
      12.0        1
      13.0        1
      14.0        1
      15.0        1
      16.0        1
      17.0        1
      18.0        1
      19.0        1
      20.0        1
      21.0        1
      22.0        1
      23.0        1
      24.0        1
      25.0        1
      26.0        1
      27.0        1
      28.0        1
      29.0        1
      30.0        1

         F   no.levels (incl. degeneracies)
       0.0        1
       1.0        1
       2.0        1
       3.0        1
       4.0        1
       5.0        1
       6.0        1
       7.0        1
       8.0        1
       9.0        1
      10.0        1
      11.0        1
      12.0        1
      13.0        1
      14.0        1
      15.0        1
      16.0        1
      17.0        1
      18.0        1
      19.0        1
      20.0        1
      21.0        1
      22.0        1
      23.0        1
      24.0        1
      25.0        1
      26.0        1
      27.0        1
      28.0        1
      29.0        1
      30.0        1

   selected M quanta: [-30, -29, -28, -27, -26, -25, -24, -23, -22, -21, -20, -19, -18, -17, -16, -15, -14, -13, -12, -11, -10, -9, -8, -7, -6, -5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30]

   total number of basis functions: 961

   Initialize Fortran TDSE module ...
   ... done


Add tensor operator

   Matrix element files template: ../cmi-richmol_watie/ocs_matelem_mu_j<j1>_j<j2>.rchm

   Rank: 1
   Index symmetry: none
   Number of unique Cartesian components: 3
     1    x      1
     2    y      1
     3    z      1

   Scan matrix element files
       F1     F2 transposed?    filename
      0.0    1.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j0_j1.rchm
      1.0    0.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j0_j1.rchm
      1.0    2.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j1_j2.rchm
      2.0    1.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j1_j2.rchm
      2.0    3.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j2_j3.rchm
      3.0    2.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j2_j3.rchm
      3.0    4.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j3_j4.rchm
      4.0    3.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j3_j4.rchm
      4.0    5.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j4_j5.rchm
      5.0    4.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j4_j5.rchm
      5.0    6.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j5_j6.rchm
      6.0    5.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j5_j6.rchm
      6.0    7.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j6_j7.rchm
      7.0    6.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j6_j7.rchm
      7.0    8.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j7_j8.rchm
      8.0    7.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j7_j8.rchm
      8.0    9.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j8_j9.rchm
      9.0    8.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j8_j9.rchm
      9.0   10.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j9_j10.rchm
     10.0    9.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j9_j10.rchm
     10.0   11.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j10_j11.rchm
     11.0   10.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j10_j11.rchm
     11.0   12.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j11_j12.rchm
     12.0   11.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j11_j12.rchm
     12.0   13.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j12_j13.rchm
     13.0   12.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j12_j13.rchm
     13.0   14.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j13_j14.rchm
     14.0   13.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j13_j14.rchm
     14.0   15.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j14_j15.rchm
     15.0   14.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j14_j15.rchm
     15.0   16.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j15_j16.rchm
     16.0   15.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j15_j16.rchm
     16.0   17.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j16_j17.rchm
     17.0   16.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j16_j17.rchm
     17.0   18.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j17_j18.rchm
     18.0   17.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j17_j18.rchm
     18.0   19.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j18_j19.rchm
     19.0   18.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j18_j19.rchm
     19.0   20.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j19_j20.rchm
     20.0   19.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j19_j20.rchm
     20.0   21.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j20_j21.rchm
     21.0   20.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j20_j21.rchm
     21.0   22.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j21_j22.rchm
     22.0   21.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j21_j22.rchm
     22.0   23.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j22_j23.rchm
     23.0   22.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j22_j23.rchm
     23.0   24.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j23_j24.rchm
     24.0   23.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j23_j24.rchm
     24.0   25.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j24_j25.rchm
     25.0   24.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j24_j25.rchm
     25.0   26.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j25_j26.rchm
     26.0   25.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j25_j26.rchm
     26.0   27.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j26_j27.rchm
     27.0   26.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j26_j27.rchm
     27.0   28.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j27_j28.rchm
     28.0   27.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j27_j28.rchm
     28.0   29.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j28_j29.rchm
     29.0   28.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j28_j29.rchm
     29.0   30.0       False    ../cmi-richmol_watie/ocs_matelem_mu_j29_j30.rchm
     30.0   29.0        True    ../cmi-richmol_watie/ocs_matelem_mu_j29_j30.rchm

   Tensor name: mu
   Number of irreps: 1
   Number of unique Cartesian components: 3

   Add new tensor in Fortran TDSE module ...
   Threshold for neglecting mtrix elements: 1e-12
   ... done

   Current tensor index: 1


Add electric field

   Number of time-points: 1

   Initialize field from input formula: ['0', 'edc0', '0']
   Parameters: {'edc0': 100000}

   Convert electric field from v/cm to V/m using factor: 100.0
Store stationary state no. 0 at a time = 0.0 into file wavefunc.0

Initial wavepacket

   Total number of basis functions: 961
   Number of initial coefficients: 45

   #basis func    M    F   ID ideg   coefficient
           253   -8    8    1    1   5.99872054e-08   0.00000000e+00i
           276   -7    7    1    1   1.42790727e-06   0.00000000e+00i
           300   -6    6    1    1  -2.68687791e-05   0.00000000e+00i
           302   -6    8    1    1   4.38084590e-08   0.00000000e+00i
           325   -5    5    1    1  -3.88017903e-04   0.00000000e+00i
           327   -5    7    1    1   1.04779748e-06   0.00000000e+00i
           351   -4    4    1    1   4.14209627e-03   0.00000000e+00i
           353   -4    6    1    1  -1.98439085e-05   0.00000000e+00i
           355   -4    8    1    1   3.93714104e-08   0.00000000e+00i
           378   -3    3    1    1   3.11295433e-02   0.00000000e+00i
           380   -3    5    1    1  -2.89211469e-04   0.00000000e+00i
           382   -3    7    1    1   9.47768479e-07   0.00000000e+00i
           406   -2    2    1    1  -1.54288970e-01   0.00000000e+00i
           408   -2    4    1    1   3.13113047e-03   0.00000000e+00i
           410   -2    6    1    1  -1.81149272e-05   0.00000000e+00i
           412   -2    8    1    1   3.75391668e-08   0.00000000e+00i
           435   -1    1    1    1  -4.60287339e-01   0.00000000e+00i
           437   -1    3    1    1   2.41128406e-02   0.00000000e+00i
           439   -1    5    1    1  -2.67757792e-04   0.00000000e+00i
           441   -1    7    1    1   9.11990646e-07   0.00000000e+00i
           465    0    0    1    1   7.13881353e-01   0.00000000e+00i
           467    0    2    1    1  -1.25976416e-01   0.00000000e+00i
           469    0    4    1    1   2.97045119e-03   0.00000000e+00i
           471    0    6    1    1  -1.76783588e-05   0.00000000e+00i
           473    0    8    1    1   3.70141179e-08   0.00000000e+00i
           496    1    1    1    1  -4.60287339e-01   0.00000000e+00i
           498    1    3    1    1   2.41128406e-02   0.00000000e+00i
           500    1    5    1    1  -2.67757792e-04   0.00000000e+00i
           502    1    7    1    1   9.11990646e-07   0.00000000e+00i
           526    2    2    1    1  -1.54288970e-01   0.00000000e+00i
           528    2    4    1    1   3.13113047e-03   0.00000000e+00i
           530    2    6    1    1  -1.81149272e-05   0.00000000e+00i
           532    2    8    1    1   3.75391668e-08   0.00000000e+00i
           555    3    3    1    1   3.11295433e-02   0.00000000e+00i
           557    3    5    1    1  -2.89211469e-04   0.00000000e+00i
           559    3    7    1    1   9.47768479e-07   0.00000000e+00i
           583    4    4    1    1   4.14209627e-03   0.00000000e+00i
           585    4    6    1    1  -1.98439085e-05   0.00000000e+00i
           587    4    8    1    1   3.93714103e-08   0.00000000e+00i
           610    5    5    1    1  -3.88017903e-04   0.00000000e+00i
           612    5    7    1    1   1.04779748e-06   0.00000000e+00i
           636    6    6    1    1  -2.68687791e-05   0.00000000e+00i
           638    6    8    1    1   4.38084611e-08   0.00000000e+00i
           661    7    7    1    1   1.42790727e-06   0.00000000e+00i
           685    8    8    1    1   5.99872062e-08   0.00000000e+00i

   Normalize coefficient vector by 1/sqrt( 1.000000000000004 )
